moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -80,7 +81,7 @@ export const ProjectsListViewQueryConf = {
           <RouterLink
             to={`/${jsonValues.singleItem}/${jsonValues.link}/`}
             state={jsonValues.state}
-            className={"MuiTypography-colorPrimary"}
```

Comment:
class name no longer exists. this prevents the visited links from appearing purple